### ISSUE TITLE
Not working in Vue 2.7

### ISSUE THREAD

**Issue by joy-yu** [ecomfe/vue-echarts#633]
### Confirmation

- [x] I can confirm this problem is not reproducible with ECharts itself.

### How are you introducing Vue-ECharts into your project?

`<script> tag`

### Versions

```js
<script src="https://cdn.jsdelivr.net/npm/vue@2.7.0"></script>
<script src="https://cdn.jsdelivr.net/npm/@vue/composition-api@1.6.2"></script>
<script src="https://cdn.jsdelivr.net/npm/echarts@5.3.3"></script>
<script src="https://cdn.jsdelivr.net/npm/vue-echarts@6.1.0"></script>
```

### Details

Not working in Vue 2.7

### Reproduction

https://codesandbox.io/s/stupefied-kate-pbj0hr?file=/index.html

**Comment by pedrolamas** [comment]
I just experienced this issue myself, can confirm that Vue 2.7 has broken this component

**Comment by Justineo** [comment]
Maybe we need to update the dependency for `vue-demi`. I’ll look into this next week.

**Comment by pedrolamas** [comment]
@Justineo seems you are correct!

I removed `@vue/composition-api` as that is not required for Vue 2.7, and then added `vue-demi@0.13.1`, ending up with this:

```html
<script src="https://cdn.jsdelivr.net/npm/vue@2.7.0"></script>
<script src="https://cdn.jsdelivr.net/npm/vue-demi@0.13.1"></script>
<script src="https://cdn.jsdelivr.net/npm/echarts@5.3.3"></script>
<script src="https://cdn.jsdelivr.net/npm/vue-echarts@6.1.0"></script>
```

And now it works again!

 https://codesandbox.io/s/keen-hertz-63qq75?file=/index.html

**Comment by Dhalsimzhao** [comment]
![image](https://user-images.githubusercontent.com/3771987/177297202-9dd6bd30-4d7b-4a1c-8313-5de633add34d.png)

Same problem happen to me, @pedrolamas could you please update the vue-demi dependency version to ^0.13.2 ?

**Comment by pedrolamas** [comment]
@Dhalsimzhao I've just updated the vue-demi dependency version to ^0.13.2 in sandbox link above as requested

**Comment by joy-yu** [comment]
@fanti1 I keep the Vue version at 2.6.x in package.json temporarily.

`"vue": "~2.6.11"`

**Comment by nba1234228** [comment]
@Justineo 能赶紧修复这个问题吗

**Comment by Dhalsimzhao** [comment]
> @Dhalsimzhao I've just updated the vue-demi dependency version to ^0.13.2 in sandbox link above as requested

Really appreciate!

**Comment by Justineo** [comment]
This should now be fixed in `vue-echarts@6.2.0`.
